src/Resources/app/storefront/src/js/filter-slider-range.plugin.js

```diff
@@ -49,7 +46,7 @@ export default class FilterSliderRange extends FilterBasePlugin {
       step: this.options.price.step,
       range: {
         'min': this.options.price.min,
-        'max': this.options.price.max,
+        'max': this.options.price.max === this.options.price.min ? this.options.price.min + 1 : this.options.price.max
```

Comment:
Maybe move this to a separate method?

Reply:
Does it break the range-slider when min and max are the same?

Reply:
Yes it gives an error that min/max cannot be same.